doc/source/reference/frame.rst

```diff
@@ -234,6 +234,7 @@ Reshaping, sorting, transposing
    DataFrame.unstack
    DataFrame.swapaxes
    DataFrame.melt
+   wide_to_long
```

Comment:
this isn't a DataFrame method. so not sure that it should be added here.

Reply:
Resolved in be23734d1a6db355e3aaaa993327901a0cb4168e